crates/atuin/src/command/client.rs

```diff
@@ -120,9 +121,20 @@ impl Cmd {
     ) -> Result<()> {
         let filter =
             EnvFilter::from_env("ATUIN_LOG").add_directive("sqlx_sqlite::regexp=off".parse()?);
-
+        let log_dir = atuin_common::utils::state_dir().join("logs");
+        // Ensure the log directory exists
+        tokio::fs::create_dir_all(&log_dir)
+            .await
+            .wrap_err("could not create log directory")?;
+        let appender = RollingFileAppender::builder()
```

Comment:
This seemed like a decent idea so the log wouldn't grow forever, but we can also just do a plain file as well